apps/docs/src/index.data.ts

```diff
@@ -0,0 +1,245 @@
+import fs from "node:fs";
+import path from "node:path";
+import { fileURLToPath } from "node:url";
+import { defineLoader } from "vitepress";
+import type { ComponentGridProps } from "./.vitepress/components/ComponentGrid.vue";
+import type { Tab } from "./.vitepress/components/TabGroup.vue";
+
+/**
+ * Build-time data for the home page (components, facts/numbers etc.)
+ */
+export type Data = {
+  /** Total number of implemented Onyx components. */
+  componentCount: number;
+  /** Total number of component variants/stories across all implemented components as documented with Storybook. */
+  variantCount: number;
+  /** Total number of merged pull requests on GitHub. */
+  mergedPRCount: number;
+  /** Total number of closed issues on GitHub. */
+  closedIssueCount: number;
+  /** Total number of GitHub contributors. */
+  contributorCount: number;
```

Comment:
could we make this Promise<number> so the rest of the app loads faster?

If I change something in the code to try smth out, the whole app freezes for a few seconds..
maybe the github API is not too happy to be requested as often and the livereload pushes the limits :(

Reply:
Its already a promise because the data is build-time so there is no way of making this faster except disabling all data fetching when running locally

Reply:
this comment got obsolete when you implemented the dev mode behavior, we can just close it :)